lib/plugins/aws/info/display.js

```diff
@@ -43,13 +44,14 @@ module.exports = {
       apiKeysMessage += '\n  None';
     }
 
-    this.serverless.cli.consoleLog(apiKeysMessage);
+    legacy.consoleLog(apiKeysMessage);
     return apiKeysMessage;
   },
 
   displayEndpoints() {
     const info = this.gatheredData.info;
-    let endpointsMessage = `${chalk.yellow('endpoints:')}`;
+    let endpointsMessageLegacy = `${chalk.yellow('endpoints:')}`;
```

Comment:
After we have outputs registration interface, I think there's no need for any changes in this file (aside of switching to legacy interface)

Reply:
leftover from previous iteration, changed :+1: